examples/okta_policy_mfa/basic_updated.tf

```diff
@@ -3,7 +3,7 @@ data "okta_group" "all" {
 }
 
 resource "okta_policy_mfa" "test" {
-  name            = "testAcc_replace_with_uuid"
+  name            = "testAcc_replace_with_uuid_new"
```

Comment:
Just cuious what the reason for this change is

Reply:
I needed to make sure that the name will change.